spec/src/main/asciidoc/core/implementation.asciidoc

```diff
@@ -757,3 +757,24 @@ A bean type must be proxyable if an injection point resolves to a bean:
 
 Otherwise, the container automatically detects the problem, and treats it as a deployment problem.
 
+[[allowProxying]]
+
+==== Explicitly allow proxying of a bean type with final methods
+
+The container has multiple ways to define a list of classes which should be considered proxyable despite of having final methods:
+
+* a comma (`','`) separated list of class names set via the java system property `javax.enterprise.inject.allowProxying.classes`, plus
+* a comma (`','`) separated list of class names set via the environment `JAVAX_ENTERPRISE_INJECT_ALLOWPROXYING_CLASSES`, plus
+* a `allowProxying` entry in `beans.xml` as shown in the following example
+
+[source, xml]
+----
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee"
```

Comment:
txs, will add the changes as well.
Will take till tomorrow afternoon as I’m hanging on some customer stuff right now :/

LieGrue,
strub


> Am 17.01.2016 um 22:33 schrieb Mark Paluch <notifications@github.com>:
> 
> In spec/src/main/asciidoc/core/implementation.asciidoc:
> 
> > @@ -757,3 +757,24 @@ A bean type must be proxyable if an injection point resolves to a bean:
> >  
> >  Otherwise, the container automatically detects the problem, and treats it as a deployment problem.
> >  
> > +[[allowProxying]]
> > +
> > +==== Explicitly allow proxying of a bean type with final methods
> > +
> > +The container has multiple ways to define a list of classes which should be considered proxyable despite of having final methods:
> > +
> > +* a comma (`','`) separated list of class names set via the java system property `javax.enterprise.inject.allowProxying.classes`, plus
> > +* a comma (`','`) separated list of class names set via the environment `JAVAX_ENTERPRISE_INJECT_ALLOWPROXYING_CLASSES`, plus
> > +* a `allowProxying` entry in `beans.xml` as shown in the following example
> > +
> > +[source, xml]
> > +----
> > +<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee"
> 
> What about the beans_2_0.xsd?
> 
> —
> Reply to this email directly or view it on GitHub.
>